### PR TITLE
Document custom plan policy inputs

### PR DESCRIPTION
**Pull request by marcinwyszynski**
## Checklist

Please make sure that the proposed change checks all the boxes below before requesting a review:

- [x] I have reviewed the [guidelines for contributing](https://github.com/spacelift-io/user-documentation/blob/main/CONTRIBUTING.md) to this repository.
- [x] The preview looks fine.
- [x] The tests pass.
- [x] The commit history is clean and meaningful.
- [x] The pull request is opened against the `main` branch.
- [x] The pull request is no longer marked as a draft.
- [x] You agree to license your contribution under the [MIT license](https://github.com/spacelift-io/user-documentation/blob/main/LICENSE) to Spacelift.

If the proposed change is ready to be merged, please request a review from `@spacelift-io/solutions-engineering`. Someone will review and merge the pull request.

Thank you for your contribution! 🙇

### CODE REVIEW

File: docs/concepts/policy/terraform-plan-policy.md [hunk 1]
```diff
@@ -135,6 +135,30 @@ deny["must not target the forbidden endpoint: forbidden.endpoint/webhook"] {
 }
 ```
 
+## Custom inputs
+
+Sometimes you might want to pass some additional data to your policy input. For example, you may want to pass the `configuration` data from the Terraform plan, the result of a third-party API or tool call. You can do that by generating a JSON file with the data you need at the root of your project. The file name must follow the pattern `$key.custom.spacelift.json` and must represent a valid JSON _object_. The object will be merged with the rest of the input data, as `input.third_party_metadata.custom.$key`. Be aware that the file name is case-sensitive. Below are two examples, one exposing Terraform configuration and exposing the result of a third-party security tool.
```
Comment (cube2222) [inline docs/concepts/policy/terraform-plan-policy.md:140]: > one exposing Terraform configuration and exposing the result of a third-party security tool

I think this is missing a word or two.

`and the other`?

**Review by cube2222 — DISMISSED** [review]
with nit

File: docs/concepts/policy/terraform-plan-policy.md (hunk 1 above)
Comment (marcinwyszynski) [reply docs/concepts/policy/terraform-plan-policy.md]: Thanks, fixed.

**Review by cube2222 — APPROVED** [review]

**Review by TheMacies — APPROVED** [review]

File: .vscode/settings.json [hunk 2]
```diff
@@ -0,0 +1,10 @@
+{
+    "cSpell.words": [
```
Comment (jmfontaine) [inline .vscode/settings.json:2]: If the goal is to enforce the correct case, there is already a [CI check for this](https://github.com/spacelift-io/user-documentation/blob/main/.markdownlint.yaml#L22).

Comment (marcinwyszynski) [reply .vscode/settings.json:2]: It's just for my local spellchecker.

**Pull request merged**